activestorage/app/models/active_storage/blob.rb

```diff
@@ -34,7 +34,7 @@ class ActiveStorage::Blob < ActiveStorage::Record
   ##
   # :method:
   #
-  # Returns the associated <tt>ActiveStorage::Attachment</tt>s.
+  # Returns the associated +ActiveStorage::Attachment+s.
```

Comment:
```suggestion
  # Returns the associated ActiveStorage::Attachment s.
```
Would this be the right way to link but also make it plural?

Reply:
I think we can just use a normal link in this case